docusaurus.config.js

```diff
@@ -248,6 +248,26 @@ module.exports = {
             spec: 'openapi/swagger.json',
```

Comment:
```suggestion
            spec: 'openapi/swagger-v2.11.json',
```

Our "latest" (`/docs` dir) version is a rolling version and content-wise it corresponds to the most recent released version, which means it's a mirror of `/version-2.11` at this time. It's purpose is to provide a "stable" link to use. Sorry for not clearing this up in my earlier comment.

Reply:
Updated the PR so that docs/ points to v2.11 swagger and the right 2.11 ID.

Removed swagger.json.